src/content/nl/2020/fonts.md

```diff
@@ -0,0 +1,361 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT2Q4hcDGGdclJH2ym0Pp_f8JWvYur_OQFQNkuScJyO7_ZCR1KPZsewL-mEZhxcuRFcde_Mxio8z_8P/pubchart?oid=919344016&format=interactive",
+  sheets_gid="1925210751",
+  sql_file="04_05.web_font_usage_breakdown_with_fcp.sql"
+  )
+}}
+
+Het was interessant om de verschillen in snelheid op te merken voor sites die de verschillende gratis / open source en commerciële services gebruiken. Als we kijken naar de tijden van First Content Paint (FCP) en Last Content Paint (LCP), bevinden sites die Google Fonts gebruiken ongeveer in het midden, maar over het algemeen iets langzamer dan de mediaan. De snelste sites in de dataset zijn Shopify en Wix (die middelen van `parastorage.com` bedienen), en er mag worden aangenomen dat ze zich richten op een klein aantal sterk geoptimaliseerde bestanden. Aan de andere kant levert Google ook weblettertypen wereldwijd van sterk variërende grootten (vanwege taal), wat waarschijnlijk resulteert in iets langzamere mediane tijden.
+
+{{ figure_markup(
+  image="fonts-median-lcp-of-sites-using-hosted-fonts.png",
+  caption="Mediaan LCP van sites die gehoste lettertypen gebruiken.",
+  description="Een staafdiagram met de gemiddelde FCP van desktop- en mobiele sites met verschillende fonthosts. `cdn.shopify.com` is het snelst met 3.335 op desktop en 8.401 op mobiel, `fonts.shopifycdn.com` staat respectievelijk op 3.224 en 8.531, `netdna.bootstrapcdn.com` op 3.910 en 8.183, `maxcdn.bootstrapcdn.com` op 4.240 en 8.530, `cdnjs.cloudflare.com` op 4.105 en 8.730, `use.fontawesome.com` op 4.519 en 9.166, `fonts.gstatic.com` op 4.878 en 9.558, `cdn.jsdelivr.net` op 5.368 en 10.646, `static.parastorage.com` staat op 4.322 en 11.813, `use.typekit.net` staat op 4.700 en 12.552.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT2Q4hcDGGdclJH2ym0Pp_f8JWvYur_OQFQNkuScJyO7_ZCR1KPZsewL-mEZhxcuRFcde_Mxio8z_8P/pubchart?oid=2012634758&format=interactive",
+  sql_file="04_05.web_font_usage_breakdown_with_fcp.sql"
+  )
+}}
+
+Bij het bekijken van commerciële services zoals Adobe (`use.typekit.net`) of Monotype (`fast.fonts.com`) is het interessant om op te merken dat ze op desktop net zo snel of iets sneller zijn dan Google Fonts, maar merkbaar langzamer op mobiel. De conventionele wijsheid heeft over het algemeen aangenomen dat de trackingscripts die door die services worden gebruikt, ze aanzienlijk vertragen, maar dat is blijkbaar tegenwoordig minder een probleem dan in de afgelopen jaren. Hoewel het waar is dat we de prestaties van de site meten en niet noodzakelijk de prestaties van de font-hoster, hebben deze trackingscripts invloed op het laden van lettertypen op de client, dus het lijkt relevant om deze observaties op te nemen.
```

Comment:
Did you see I changed this in #1933? Wasn't happy with the word "font hoster" so changed this to:

> While it's true that we are measuring site performance and not necessarily performance of the font host, those tracking scripts impact font loading on the client so it seems relevant to include these observations.

Not sure if you want to make a similar change here?